ccmlib/scylla_node.py

```diff
@@ -38,6 +39,52 @@
 if TYPE_CHECKING:
     from ccmlib.scylla_cluster import ScyllaCluster
 
+class ScyllaKeyType:
+    """
+    Helper class to define Scylla Key definition with Type strings.
+    Use the cassandra class name from https://github.com/scylladb/scylladb/blob/master/docs/dev/cql3-type-mapping.md as type strings.
+    Examples :
+        type1 = ScyllaKeyType.make_partition_key("Int32Type")
+        type2 = ScyllaKeyType.make_clustering_key("Int32Type", "FloatType")
+    """
+    class TypeKind(Enum):
+        REGULAR_KEY = 1
+        CLUSTERING_KEY = 2
+        PARTITION_KEY = 3
+
+    kind: TypeKind
+    types: Tuple[str]
```

Comment:
Does this need to be tuple, from the usages it seems you can use `Iterable[str]` instead?

Reply:
Yes, I have replaced it now.

Reply:
Neither the iterable from typing nor the Iterable from collections.abc work in 3.8. So, I have changed it back to Tuple[str]. Is that okay?

Reply:
I think it should work https://docs.python.org/3.8/library/typing.html#typing.Iterable, but I am fine with leaving Tuple in as well

Reply:
It worked! Thank you, I have replaced it.